model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -78,6 +78,7 @@ data class PackageCurationData(
                     type = vcs.type.takeUnless { it.isBlank() } ?: pkg.vcs.type,
                     url = vcs.url.takeUnless { it.isBlank() } ?: pkg.vcs.url,
                     revision = vcs.revision.takeUnless { it.isBlank() } ?: pkg.vcs.revision,
+                    resolvedRevision = vcs.resolvedRevision.takeUnless { it == null } ?: pkg.vcs.resolvedRevision,
```

Comment:
How about simplifying this to `vcs.resolvedRevision ?: pkg.vcs.resolvedRevision` (even if it breaks the `takeUnless()` pattern with the surrounding lines)?

Reply:
Done.